### ISSUE TITLE
piusrc doesn't handle `~` for paths

### ISSUE THREAD

**Issue by philpennock** [jaymzh/pius#42]
For a config file, tilde-expansion needs to be handled when reading the file instead of leaving it to the shell.

Desired to work in `~/.pius/piusrc`:

```
mail-text=~/.pius/mail-template
```

This lets the config be checked into revision control and checked out in multiple environments.

**Comment by philpennock** [comment]
Forgot version, sorry:

```
% pius --version
pius 2.2.0
% lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.5 LTS
Release:        14.04
Codename:       trusty
```

**Comment by jaymzh** [comment]
Fixed by #108
